### DISCUSSION TITLE
Cataclysm water texture packs

### DISCUSSION THREAD

**Discussion by Peperss** [chromiecraft/chromiecraft#2185] [General]
hey guys do you know if there is a way to implement the improvements made in the cataclysm expansion regarding water effects? Or if someone is already working on creating a patch for the community?

**Comment by VFaleiro** [comment]
I've found [this](http://www.modcraft.io/index.php?topic=8253.0).
I have not tested it so I'm not sure it works.
Use it at your own risk by the way.
